modules/pel/peltool/peltool.py

```diff
@@ -201,6 +201,13 @@ def prettyPrint(Mdata: str, desiredSpace: int = 34) -> str:
     return '\n'.join(lines)
 
 
+def checkSeverity(uh: UserHeader, config: Config) -> bool:
```

Comment:
Please add function document.

Reply:
Also, maybe make the function name a bit more specific, because at first I was thinking it would return true if there was a match, not false.